src/api/listNFTs.ts

```diff
@@ -1,9 +1,23 @@
 import {Currency} from '@tatumio/tatum';
-// eslint-disable-next-line @typescript-eslint/no-unused-vars
-import {TATUM_CUSTOM_CONTRACT_ADDRESS, API_KEY} from '../utils/constants';
+import {TATUM_API_KEY} from '../utils/constants';
 import axios from 'axios';
 import {INFTResponse} from './@types/NFTResponse';
 
+function handleParseMetadata(metadata: any) {
+    return metadata.map((itemMetadata: any) => {
+        return {
+            url: itemMetadata.url,
+            metadata:
+                itemMetadata.metadata != null
+                    ? itemMetadata.metadata.name != null
+                        ? itemMetadata.metadata
```

Comment:
This validation is already being made in the first if. Why do you use != instead of !==?

Reply:
Given the different choices we've implemented on this over time (and some minted nfts with empty data) I am using != to check for null and undefined. A strict comparison for name here would break if an object has metadata equal to null (which is happening on some of the minted nfts)
In the future, once the contract has been switched, we can change this